.github/workflows/test.yml

```
@@ -0,0 +1,34 @@
+name: Test
+
+on:
+  workflow_call:
+    inputs:
+      node_versions:
+        required: true
+        type: string
+
+jobs:
+  test_matrix:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node_version:
+          - 14
+          - 16
+          - 18
+    steps:
+      - uses: actions/checkout@2541b1294d2704b0964813337f33b291d3f8596b # tag=v3
+      - name: "Test with Node.js ${{ matrix.node_version }}"
+        uses: actions/setup-node@5b949b50c3461bbcd5a540b150c368278160234a # tag=v3
+        with:
+          node-version: "${{ matrix.node_version }}"
+          cache: npm
+      - run: npm ci
+      - run: npm run test --ignore-scripts # run lint only once
+  test:
```

What do you think of renaming this job to be more indicative?

```suggestion
  lint:
```

---

it's not just linting, we also test things that are independent of node versions, e.g. type tests

---

Interesting! Does thé `lint` npm script do all of those things? (I made this suggestion because the only substantive command in the steps is to run the `lint` script.)